modules/cluster-logging-collector-log-forward-syslog.adoc

```diff
@@ -96,6 +96,49 @@ spec:
 $ oc create -f <file-name>.yaml
 ----
 
+[id=cluster-logging-collector-log-forward-examples-syslog-log-source]
+== Adding log source information to message output
+
+You can add `namespace_name`, `pod_name`, and `container_name` elements to the `message` field of the record by adding the `AddLogSource` field to your `ClusterLogForwarder` custom resource (CR).
+
+[source,yaml]
+----
+  spec:
+    outputs:
+    - name: syslogout
+      syslog:
+        addLogSource: true
+        facility: user
+        payloadKey: message
+        rfc: RFC3164
+        severity: debug
+        tag: mytag
+      type: syslog
+      url: tls://syslog-receiver.openshift-logging.svc:24224
+    pipelines:
+    - inputRefs:
+      - application
+      name: test-app
+      outputRefs:
+      - syslogout
+----
+
+[NOTE]
+====
+This configuration is compatible with both RFC3164 and RFC5424.
+====
+
+Syslog message output without `AddLogSource`:
+----
+<15>1 2020-11-15T17:06:14+00:00 fluentd-9hkb4 mytag - - -  {"msgcontent"=>"Message Contents", "timestamp"=>"2020-11-15 17:06:09", "tag_key"=>"rec_tag", "index"=>56}
+----
+.Example syslog message output with `AddLogSource`
+
+[source,json]
```

Comment:
I don't think this should have the [source,json] formatting, because this is what it looks like in the output.
![Highlighting](https://user-images.githubusercontent.com/4106682/169117790-07427f93-9e07-4737-affa-d6c8dc06d018.png)

Reply:
It contains JSON but also has a wacky rsyslog prefix, I doubt there's a highlighter that will get it right. Just [source] is probably the best you can do.